app/models/concerns/permit_application_status.rb

```diff
@@ -0,0 +1,57 @@
+module PermitApplicationStatus
+  extend ActiveSupport::Concern
+  included do
+    include AASM
+    enum status: { new_draft: 0, newly_submitted: 1, revisions_requested: 3, resubmitted: 4 }, _default: 0
```

Comment:
yeah that's what I was trying to push forward, but the mocks need to show if a permit application was revision requested, or resubmitted.
Originally I was determining this using timestamps, but Sunny and Laurel did not let me put this through and said I needed to refactor this with the new states, leading to these somewhat redundant states.
The reason new_draft and newly_submitted is to distinguish it from the other draft and submitted states respectively. This is so that these are not confused with the filters on the front end which do not correspond to actual statuses, but status groups.

Reply:
ok. Personally like the timestamp approach if the only need is to show that  data has been resubmitted, but I don't have enough context so assuming it has been discussed in detail already.